.github/workflows/release.yml

```
@@ -0,0 +1,32 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - uses: pnpm/action-setup@v4
+        with:
+          version: latest
+      - run: pnpm install --frozen-lockfile
+      - run: pnpm run build
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)